### ISSUE TITLE
Use in-app camera for profile photo

### ISSUE THREAD

**Issue by angelo-fuchs** [deltachat/deltachat-android#442]
When creating / selecting a photo for the profile picture one can select various camera options, but in the chat view we already have a built-in camera and should offer that use here as well instead of switching to a different app.

**Comment by r10s** [comment]
of course, this would be nice to have, however, it may also be quite some effort. not sure if this is worth for a function typically used only once or very few times.

btw, just noticed the group-profile-image-button offers different functions than the user-profile-image-button. maybe we should start consolidating this first. also as changes in one area will affect the other one then, too :)
